.changeset/neat-masks-travel.md

```diff
@@ -0,0 +1,5 @@
+---
+"@bigcommerce/catalyst-core": minor
+---
+
+Add support for logged-in customer checkout using Customer Login API
```

Comment:
I think we need to add more information for users now that we are requiring the `BIGCOMMERCE_CLIENT_SECRET` and `BIGCOMMERCE_CLIENT_ID` to be required here. There will also need to be some changes to the CLI in order for new initializations to be using this.

Reply:
I did add some new docs, but I can add the callout to the changelog also.

The CLI can't autoconfigure this for $reasons - merchants need to create an explicit API Account in the CP in order to get a client secret, the CLI has no ability to do that automatically.

Reply:
That's making me think that maybe we don't do this change if it's required to include those environment variable. As much as this would make things a bit better from the storefront perspective, impeding the flow for creating a Catalyst app might be detrimental to devs.